mod_ci/controllers.py

```diff
@@ -568,6 +569,36 @@ def start_ci():
                 issue.status = issue_data['state']
                 g.db.commit()
 
+        elif event == "release":
+            release_data = payload['release']
+            release_type = payload['prerelease']
+            # checking whether it is meant for production
+            if release_type is False:
+                return
+            else:
+                release_version = payload['tag_name']
+                # Github recommends adding v to the version
+                if release_version[0] == 'v':
+                    release_version = release_version[1:]
+                release_date = payload['created_at']
+                release_commit = GeneralData.query.filter(GeneralData.key == 'last_commit').first()
+                release = CCExtractorVersion(release_data, release_version, release_commit)
+                g.db.add(release)
+                g.db.commit()
+                # adding test corresponding to last commit to the baseline regression results
+                test = Test.query.filter(Test.commit == release_commit).first()
+                test_result_file = g.db.query(TestResultFile).filter(
+                                    TestResultFile.test_id == test.id).subquery()
+                test_result = g.db.query(TestResult).filter(
+                                    TestResult.test_id == test.id).subquery()
+                g.db.query(RegressionTestOutput.correct).filter(
+                                RegressionTestOutput.regression_id == test_result_file.c.regression_test_id &
```

Comment:
You'll have to wrap the sections in parenthesis if you want to use `&` according to the docs: http://docs.sqlalchemy.org/en/latest/core/sqlelement.html#sqlalchemy.sql.expression.and_

Reply:
i forgot it

Reply:
i will add